src/Liquid/AbstractBlock.php

```diff
@@ -164,7 +164,7 @@ protected function renderAll(array $list, Context $context)
 			}
 
 			if (is_array($value)) {
-				throw new RenderException("Implicit rendering of arrays not supported. Use index operator.");
+				$value = print_r($value, true);
```

Comment:
How about wrapping this with `htmlspecialchars`? I'm concerned we might be adding XSS here where folks had a plain error.

Reply:
Sounds good- will commit.

Reply:
Ah, so print_r shows some HTML?

Reply:
```
Array
(
    [0] =&gt; a
    [1] =&gt; b
    [2] =&gt; c
)```